src/main/openapi/discovery.yaml

```diff
@@ -250,9 +250,19 @@ components:
         contact:
           $ref: "#/components/schemas/ValueLabel"
           title: Contact
+        hasVersions:
+          type: array
+          items:
+            $ref: "#/components/schemas/ValueLabel"
+          title: Has Versions
         accessRights:
           $ref: "#/components/schemas/ValueLabel"
           title: Access Rights
+        conformsTo:
```

Comment:
important: the field should be marked as deprecated, please check the OpenAPI 3.x specification if there is a way to do that.